typescript/cli/examples/submit/strategy/gnosis-ica-strategy.yaml

```diff
@@ -0,0 +1,13 @@
+chain: avalanche
+submitter:
+  type: gnosisSafe
+  props:
+    chain: avalanche
+    safeAddress: '0x7fd32493Ca3A38cDf78A4cb74F32f6292f822aBe'
+transforms:
+  - type: interchainAccount
+    props:
```

Comment:
also can we remove the `props` key from transforms and submitter?
imo theres no reason to have 3 layers of config here

Reply:
fasho– used `...ZodSchema.shape` to do so